docs/API.md

```diff
@@ -76,6 +76,21 @@ The json will be:
   ]
 }
 ```
+
+# GET - video informations
+
+Route: `GET /api/v1/videos/{id}/info`
+
+Json video informations
+
+The json will be:
+
+```json
+{
+  "title": "title",
+  "uploaddate": "date",
```

Comment:
CamerCase ?

Reply:
Camelcase was not used in the rest of the file (ex : line 61-63). Should I still correct it ?